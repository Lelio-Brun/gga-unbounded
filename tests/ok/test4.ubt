group_setting 1.

oracle o1(m:Fp) =
  return [] in G1.

win (a:Fp, b:Fp, c:Fp) =
  ( forall i: (sum j: a_j) = b_i  /\
    (sum i: a_i) = c
	).

proof.
  
simplify.

