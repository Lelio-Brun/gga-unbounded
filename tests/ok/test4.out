Working on goal 1 out of 1.
  1. (sum k'1: a_k'1) - c = 0
  2. forall j'1. a_j'1 + (sum k'1 <> {j'1}: a_k'1) - b_j'1 = 0
